docs/wiki/Fetching latest changes - Scenario 1 Module library only.md

```diff
@@ -0,0 +1,99 @@
+When the CARML library is consumed with the [*Template Orchestration*](./Solution%20creation#template-orchestration) approach modules need to be stored in an available location, where the Azure Resource Manager (ARM) can access them. This can be achieved by storing the modules templates in an accessible location like local, Template Specs or the Bicep Registry.
+
+---
+### _Navigation_
+
+- [Local Repository](#local-repository)
+- [Template Specs](#template-specs)
+- [Bicep Registry](#bicep-registry)
+
+---
+
+# Local Repository
+In case you consume the module library with a local repository, the update procedure has the following steps:
+1. Backup your local repository to maintain a backup copy
+    <details>
+    <summary>Rename your local repository</summary>
+
+    Assuming the local repository location is `'D:\ResourcesModules'` rename it in `'D:\ResourcesModules_Backup'`.
+    The backup folder can be used to compare customized modules with the ones coming from the latest version and implement the required changes.
+    This can be done, for example, by the `'Compare selected'` [function](https://vscode.one/diff-vscode/) of Visual Studio Code
+    </details>
+
+1. [Download the library](./Getting%20started%20-%20Scenario%202%20Consume%20library#1-download-the-library)
+1. [(Optional) Convert library to ARM](./Getting%20started%20-%20Scenario%202%20Consume%20library#2-optional-convert-library-to-arm)
+
+If you have customized modules, you'll need to apply customizations on the new updated modules.
```

Comment:
This is the main reason for the `fetching latest updates` section, hence I'd suggest to elaborate more on this point. e.g. highlight conflicts are expected and can be minimized by contributing back to the main repository if changes are useful for the public.

Reply:
added customization explanation